README.md

```diff
@@ -121,10 +121,10 @@ The extension provides access to several Terraform commands through the Command
 
 The Terraform VS Code extension bundles the [Terraform Language Server](https://github.com/hashicorp/terraform-ls) and is a self-contained install.
 
-The extension does require the following to be installed before use:
+The extension recommends the following to be installed before use
```

Comment:
I hate to be nitpicky here, but in line with Radek's suggestion using 'recommends' implies optional. We require 1.75 after this, so it's not optional. One could argue that terraform is optional in the sense that most functionality works, but we do not currently tell the use what doesn't work when it's not there. I would prefer require remain.

Reply:
Okay, let's leave everything as it is